README.md

```diff
@@ -244,11 +244,11 @@ If you configure your installation to have the React SDK static content served f
 If the `MediaCo` app was imported to your Infinity server, a `MediaCoOAuth` OAuth 2.0 Client Registration record will have been imported as well. That record's clientId is currently referenced within sdk-config.json.  However, you can create your own OAuth 2.0 Client Registration record using the following procedure:
    * Create a new "Security/OAuth 2.0 Client Registration" record for your app
    * You might name it the same name as your applicaion
-   * Specify "Public" for the type of client (as browser apps are not able to prevent any "Client secret" from being compromised)
+   * Specify "Public" for the type of client
    * Select "Authorization Code" for the Grant type
    * Add a RedirectURI value based on the url used to access the deployed React SDK (e.g., http://localhost:3502/auth.html)
```

Comment:
Add RedirectURI values based on what was specified above:  auth.html as well as ones for any non embedded scenarios you want to directly access via a url.  For non-embedded portal scenarios, it will redirect back to that same portal page route.